jaxtyping/_decorator.py

```diff
@@ -225,15 +225,12 @@ def f(...): ...
     """
 
     global _tb_flag
-    if (
-        _tb_flag
-        and importlib.util.find_spec("jax") is not None
-        and importlib.util.find_spec("jaxlib") is not None
-        and importlib.util.find_spec("jax._src.traceback_util") is not None
-    ):
-        import jax._src.traceback_util as traceback_util
-
-        traceback_util.register_exclusion(__file__)
+    if _tb_flag:
+        try:
+            import jax._src.traceback_util as traceback_util
+            traceback_util.register_exclusion(__file__)
+        except:
```

Comment:
Raw `except`s are pretty dangerous! This would also catch syntax errors, name errors, system exits, keyboard interrupts etc.

Can you switch this over to just `except Exception`?

Reply:
I changed it to `except ImportError` which seems like a reasonable choice.